plugins/feedback/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@janus-idp/backstage-plugin-feedback",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.4.3",
+    "@backstage/config": "^1.1.1",
+    "@backstage/core-components": "^0.13.6",
+    "@backstage/core-plugin-api": "^1.7.0",
+    "@backstage/plugin-catalog-react": "^1.9.2",
+    "@backstage/theme": "^0.4.3",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@one-platform/opc-feedback": "0.1.1-alpha",
+    "axios": "^1.6.4",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.23.0",
+    "@backstage/core-app-api": "^1.11.0",
+    "@backstage/dev-utils": "1.0.22",
+    "@backstage/test-utils": "1.4.4",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^14.0.0",
+    "msw": "^1.0.0"
+  },
+  "files": [
+    "app-config.janus-idp.yaml",
+    "dist",
+    "config.d.ts"
+  ],
+  "configSchema": "config.d.ts"
```

Comment:
```suggestion
  "configSchema": "config.d.ts",
  "repository": "github:janus-idp/backstage-plugins",
  "keywords": [
    "backstage",
    "plugin"
  ],
  "homepage": "https://janus-idp.io/",
  "bugs": "https://github.com/janus-idp/backstage-plugins/issues"
```

Reply:
Thanks, I have added these labels in package.json